### PR TITLE
UI changes: showing coding and mcq list on same page, 2 tabs

### PR DESCRIPTION
**Pull request by somnath77**
https://trello.com/c/ZwbMhlPF

### CODE REVIEW

File: src/modules/admin/createQuestion/CreateQuestionContainer/index.jsx [hunk 1]
```diff
@@ -170,7 +170,7 @@ const CreateQuestionContainer = () => {
   });
 
   const finishQuestionCreation = useCallback(() => {
-    history.push(ROUTES.ADMIN + ADMIN_ROUTES.QUESTIONS);
+    history.push(ROUTES.ADMIN + ADMIN_ROUTES.PROBLEMS);
```
Comment (Durgesh00) [inline src/modules/admin/createQuestion/CreateQuestionContainer/index.jsx:173]: avoid . operator and destructure all the routes above before using it.

File: src/modules/admin/editQuestion/editQuestionContainer/index.jsx [hunk 2]
```diff
@@ -160,7 +160,7 @@ const EditQuestionContainer = () => {
   });
 
   const finishQuestionEdit = useCallback(() => {
-    history.push(ROUTES.ADMIN + ADMIN_ROUTES.QUESTIONS);
+    history.push(ROUTES.ADMIN + ADMIN_ROUTES.PROBLEMS);
```
Comment (Durgesh00) [inline src/modules/admin/editQuestion/editQuestionContainer/index.jsx:163]: same here.

File: src/modules/admin/question/QuestionComponent/QuestionHeaderComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  totalQuestions,
+  handleAddQuestionClick,
+}) => {
+  return (
+    <div className='problem-bank-section-head d-flex align-items-center px-3'>
+      <div className='d-flex align-items-center ml-auto'>
+        <SearchBox
+          placeholder='Search Questions'
+          className='mr-3'
+          handleSearchClick={() => { }} // TODO
```
Comment (Durgesh00) [inline src/modules/admin/question/QuestionComponent/QuestionHeaderComponent.jsx:17]: is search functionality not completed?

File: src/modules/admin/questionTabs/QuestionTabsComponent/index.jsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  addDifficultyFilter,
+  filter,
+  removeDifficultyFilter,
+}) => {
+  return (
+    <Card className='bg-card mt-5'>
+      <CardHeader tag='h6' className='p-0 d-flex'>
+        <Nav className='navitem-style ml-5'>
+          <NavItem>
+            <NavLink active={activeTab === 'Coding'} onClick={() => setActiveTabMethod('Coding')}>
```
Comment (Durgesh00) [inline src/modules/admin/questionTabs/QuestionTabsComponent/index.jsx:33]: can write onClick={setActiveTabMethod('Coding')}

File: src/modules/admin/questionTabs/QuestionTabsContainer/index.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  const [showFilter, setShowFilter] = useState(false);
+  const [filter, setFilter] = useState({ difficultyFilter: '' });
+  const [searchText, setSearchText] = useState();
+  const dispatch = useDispatch();
+  const {
+    totalProblems,
+    isProblemsLoading,
+  } = useSelector((item) => item.problemsListReducer);
+  const handleAddProblemClick = () => {
+    history.push(ROUTES.ADMIN + ADMIN_ROUTES.CREATE_PROBLEM);
```
Comment (Durgesh00) [inline src/modules/admin/questionTabs/QuestionTabsContainer/index.js:30]: destructure before use

File: src/modules/admin/editQuestion/editQuestionContainer/index.jsx (hunk 2 above)
Comment (somnath77) [reply src/modules/admin/editQuestion/editQuestionContainer/index.jsx:163]: DONE

File: src/modules/admin/createQuestion/CreateQuestionContainer/index.jsx (hunk 1 above)
Comment (somnath77) [reply src/modules/admin/createQuestion/CreateQuestionContainer/index.jsx:173]: DONE

File: src/modules/admin/questionTabs/QuestionTabsContainer/index.js (hunk 5 above)
Comment (somnath77) [reply src/modules/admin/questionTabs/QuestionTabsContainer/index.js:30]: DONE

File: src/modules/admin/questionTabs/QuestionTabsComponent/index.jsx (hunk 4 above)
Comment (somnath77) [reply src/modules/admin/questionTabs/QuestionTabsComponent/index.jsx:33]: No, this is the way to pass params to method

File: src/modules/admin/question/QuestionComponent/QuestionHeaderComponent.jsx (hunk 3 above)
Comment (somnath77) [reply src/modules/admin/question/QuestionComponent/QuestionHeaderComponent.jsx:17]: No

**Pull request merged**